interface_config.js

```diff
@@ -117,7 +117,7 @@ var interfaceConfig = {
      * If indicated some of the error dialogs may point to the support URL for
      * help.
      */
-    // SUPPORT_URL: "",
+    SUPPORT_URL: 'https://github.com/jitsi/jitsi-meet/issues/new',
```

Comment:
I like this change. Not sure if that's the exact link that should be linked to, just because you need a github account to do anything and I'm not sure if people would want to make one, BUT I still really like it and it's a simple solution that should work out for now.